environments/openshift/metrics.json

```diff
@@ -1,6 +1,7 @@
 [
   "{__name__=~\"cluster:usage:.*\"}",
-  "{__name__=\"up\"}",
```

Comment:
this is something new, and we will need to fix our sync strategy for this.. we have to have the whitelist be the union of all whitelists we had previously, meaning this should still be in there, in addition to the new ones

Reply:
Yeah. I was wondering about that.